zmq-deferred/src/jbuild

```diff
@@ -0,0 +1,7 @@
+(jbuild_version 1)
+
+(library
+ ((name zmq_deferred)
+  (public_name zmq.deferred)
+  (wrapped true)
```

Comment:
I'd omit here it for consistency. Or add `(wrapped true)` elsewhere where it's implicit.

Reply:
Will do.